board/common/rootfs/etc/avahi/avahi-autoipd.action

```diff
@@ -43,7 +43,7 @@ if [ -x /bin/ip -o -x /sbin/ip ] ; then
         BIND)
             ip addr flush dev "$2" label "$2:avahi"
             ip addr add "$3"/16 brd 169.254.255.255 label "$2:avahi" scope link dev "$2" proto 6
-            ip route add default dev "$2" metric "$METRIC" scope link ||:
+            ip route add default dev "$2" metric "$METRIC" scope link proto 17 ||:
```

Comment:
Unlike `ip addr`, I think `proto zeroconf` is accepted here.

Reply:
Yes, with our patches to iproute2 that works. Not sure if it's in upstream yet, I just moved the file to common.  Great find though, thanks!

Reply:
Pretty sure `ip route ... proto <symbolic>` has worked since the olden times. `ip addr ... proto <symbolic>` was broken though.

Reply:
Ah, I'm confusing them as always, thanks!